.github/actions/install-dpcpp/action.yml

```diff
@@ -30,6 +30,7 @@ runs:
           latest=$(curl -sS https://api.github.com/repos/intel/llvm/releases | jq -r '[.[].tag_name|select(match("nightly-[0-9]{4}-[0-9]{2}-[0-9]{2}"))][0]')
           URL=https://github.com/intel/llvm/releases/download/${latest}/sycl_linux.tar.gz;
         fi
+        sudo apt install -y libhwloc15
```

Comment:
Can you move this, because right now it's sort of in the middle of installing dpcpp